### PR TITLE
Ft create ride 158538465

### PR DESCRIPTION
**Pull request by Xerrex**
### This pull request contains:
* Adding of a date validator
* Creation of a Ride Model
* Adding of the Ride Blueprint
* Data container and fetching methods
* Registration of blueprint
* Refactor Tests

#### Where the reviewer should start
* **app/validators.py**
* **app/models.py**
* **app/ride_bp/**
* **app/data/ride_data.py**
* **app/__init__.py**
* **tests/test_ride.py**

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -9,4 +9,20 @@ def __init__(self, name, email, password):
...
+    """Define the Ride Model
+    """
+    def __init__(self,**kwargs):
+        """Create a Ride Instance
+        """
+
+        self.starting_point = kwargs['starting_point']
+        self.destination = kwargs['destination']
+        self.depart_time = kwargs['depart_time']
+        self.eta = kwargs['eta']
```
Comment (JoshuaOndieki) [inline app/models.py]: What does eta represent?

Comment (Xerrex) [reply app/models.py]: Estimated time of Arrival

Comment (JoshuaOndieki) [reply app/models.py]: I'd prefer `estimated_arrival_time` over `eta`

Comment (meshack-mbuvi) [reply app/models.py]: me too, I would prefer `estimated_arrival_time` over `eta`

Comment (Ogollah) [reply app/models.py]: @JoshuaOndieki 's idea of `estimated_arrival_time` is more preferable.

Comment (TheSteelGuy) [reply app/models.py]: you guys are forgetting to say why such naming is essential. It is considered a pythonic convention to always use descriptive variable naming

**Pull request merged**